### PR TITLE
Port documentation from Wiki: ShortText Field #332

### LINKED ISSUE #332: Port documentation from Wiki: ShortText Field
Port old article from wiki and refresh them for sensenet 7. The new article should be an .md file in the /_docs folder. Old screenshots should be re-created on the new admin-ui if it is possible and placed into the /docs/images folder.
[admin.sensenet.com](https://admin.sensenet.com)
name: BuiltIn\admin
password: admin
repository url: https://dev.demo.sensenet.com

The .md file should have the following header:

```
---
title: %title%
source_url: %url of the file on github%
category: Development
version: v7.0.0
tags: %possible tags as an array%
---
```

(you can use [the docs here](https://github.com/SenseNet/sensenet.github.io/tree/master/_docs) as example)

[ShortText field](http://wiki.sensenet.com/ShortText_Field)

Please change the old wiki urls as if they were available on github (e.g. https://wiki.sensenet.com/content_templates sould be changed to /content-templates)

### PR DESCRIPTION
**Pull request by shreyasbapat**
Fixes #332

### CODE REVIEW

**Comment by shreyasbapat** [comment]
@herflis  Please review

File: _docs/shorttext-field.md [hunk 1]
```diff
@@ -2,39 +2,39 @@
...
 
-ShortText Field is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the [Field Setting](/docs/field-setting) of the Field). If you want to store text strings longer than this consider using the [LongText Field](/docs/longtext-field).
+ShortText [Field](http://wiki.sensenet.com/Field) is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the Field Setting of the Field). If you want to store text strings longer than this consider using the LongText Field.
+
+## Details
 
 The following apply to the behavior of the Field:
 
-- **Regular expression**: data input can be validated against a regular expression defined in the [Field Setting](/docs/field-setting).
+- **Regular expression**: data input can be validated against a regular expression defined in the [Field Settings](http://wiki.sensenet.com/Field_Setting).
```
Comment (herflis) [inline _docs/shorttext-field.md:20]:
```suggestion
- **Regular expression**: data input can be validated against a regular expression defined in the [Field Settings](/field-setting).
```

File: _docs/shorttext-field.md [hunk 2]
```diff
@@ -2,39 +2,39 @@
...
 tags: [fields, shorttext]
 description: ShortText Field is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the Field Setting.
 ---
 
 # ShortText Field
+ 
+## Overview
 
-ShortText Field is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the [Field Setting](/docs/field-setting) of the Field). If you want to store text strings longer than this consider using the [LongText Field](/docs/longtext-field).
+ShortText [Field](http://wiki.sensenet.com/Field) is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the Field Setting of the Field). If you want to store text strings longer than this consider using the LongText Field.
```
Comment (herflis) [inline _docs/shorttext-field.md:14]:
```suggestion
ShortText [Field](/field) is used for storing short strings. Maximum length of the stored data is 450 characters (this value however can be limited to a lower value in the Field Setting of the Field). If you want to store text strings longer than this consider using the [LongText Field](/longtext-field).
```

File: _docs/shorttext-field.md [hunk 3]
```diff
@@ -2,39 +2,39 @@
...
-</Field>
+   <Field name="ShortDesc" type="ShortText">
+   ...
+   </Field>
 ```
 
 ## Supported Field Controls
-
-- [ShortText Field Control](/docs/shorttext-fieldcontrol): a simple textbox control where user can edit value (if non-readonly).
+- [ShortText Field Control](http://wiki.sensenet.com/ShortText_Field_Control): a simple textbox control where user can edit value (if non-readonly).
```
Comment (herflis) [inline _docs/shorttext-field.md:34]:
```suggestion
- [ShortText Field Control](/shorttext-fieldcontrol): a simple textbox control where user can edit value (if non-readonly).
```

File: _docs/shorttext-field.md [hunk 4]
```diff
@@ -2,39 +2,39 @@
...
 
 ## Configuration
-
-The following properties can be set in the Field's [Field Setting](/docs/field-setting) configuration:
+The following properties can be set in the Field's [Field Setting](http://wiki.sensenet.com/Field_Setting) configuration:
```
Comment (herflis) [inline _docs/shorttext-field.md:37]:
```suggestion
The following properties can be set in the Field's [Field Setting](/field-setting) configuration:
```

File: _docs/shorttext-field.md [hunk 5]
```diff
@@ -63,11 +62,14 @@ Fully featured example:
...
 - at most 15 characters are allowed to be inserted
 - at least 3 characters are expected to be inserted
 - Field value is editable (not read-only)
-- Field can contain only letters and numbers
\ No newline at end of file
+- Field can contain only letters and numbers
+## Related links
+- [Field](http://wiki.sensenet.com/Field)
+- [ShortText Field Control](http://wiki.sensenet.com/ShortText_Field_Control)
+## References
```
Comment (herflis) [inline _docs/shorttext-field.md:75]: please remove this part

**Comment by herflis** [comment]
Hey @shreyasbapat , please commit the suggestions that I've made

**Comment by shreyasbapat** [comment]
Hi,

I am very busy for next couple of days. Will do the required commit after
that.

Thanks
Shreyas

**Comment by herflis** [comment]
Hey @shreyasbapat , please resolve the conflicts and the suggested things, because otherwise we cannot accept it 😞

**Pull request closed without merging**